packages/documentation-v7/src/stories/components/topic-teaser/topic-teaser.stories.ts

```diff
@@ -0,0 +1,185 @@
+        type: 'number',
+        min: 1,
+        max: 10,
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    alignment: {
+      name: 'Image alignment',
+      description: 'The alignment of the image.',
+      control: {
+        type: 'radio',
+        labels: {
+          'topic-teaser-reverse': 'Image left',
+          'null': 'Image right',
+        },
+      },
+      options: ['topic-teaser-reverse', 'null'],
+      table: {
+        category: 'General',
+      },
+    },
+    backgroundColor: {
+      name: 'Background Color',
+      description: 'The color the component uses as background.',
+      control: {
+        type: 'select',
+        labels: {
+          'bg-light': 'Light',
+          'bg-dark': 'Dark',
+          'bg-nightblue': 'Nightblue',
+          'bg-nightblue-bright': 'Nightblue (bright)',
+          'bg-petrol': 'Petrol',
+          'bg-petrol-bright': 'Petrol (bright)',
+          'bg-coral': 'Coral',
+          'bg-coral-bright': 'Coral (bright)',
+          'bg-olive': 'Olive',
+          'bg-olive-bright': 'Olive (bright)',
+          'bg-purple': 'Purple',
+          'bg-purple-bright': 'Purple (bright)',
+          'bg-aubergine': 'Aubergine',
+          'bg-aubergine-bright': 'Aubergine (bright)',
+        },
+      },
+      options: [
+        'bg-light',
+        'bg-dark',
+        'bg-nightblue',
+        'bg-nightblue-bright',
+        'bg-petrol',
+        'bg-petrol-bright',
+        'bg-coral',
+        'bg-coral-bright',
+        'bg-olive',
+        'bg-olive-bright',
+        'bg-purple',
+        'bg-purple-bright',
+        'bg-aubergine',
+        'bg-aubergine-bright',
+      ],
+      table: {
+        category: 'General',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+const linkTexts: string[] = [
+  'At vero eos et accusam et',
+  'Dolores et ea rebum',
+  'Stet clita kasd gubergren',
+  'Sed diam nonumy eirmod',
+  'Duo dolores et ea rebum',
+  'Magna aliquyam erat',
+  'Consetetur sadipscing elitr',
+  'Justo duo dolores',
+  'Takimata sanctus est',
+  'Nonummy nibh euismod',
+];
+
+export const Default: Story = {
+  render: (args: Args) => {
+    const classes: string = ['topic-teaser', args.backgroundColor, args.alignment, 'mb-huge-r']
+      .filter(c => c && c !== 'null')
+      .join(' ');
+
+    const links: TemplateResult[] = linkTexts.slice(0, args.linkCount).map(
+      text => html`
+        <li class="link-list-item">
+          <a href="#">
+            <span>${text}</span>
+          </a>
+        </li>
+      `,
+    );
+
+    const image = html`
+      <div key="image" class="topic-teaser-image-container">
+        <img
+          class="topic-teaser-image"
+          src="https://picsum.photos/id/553/800/800"
+          width="100%"
+          height="100%"
+          alt="Test teaser image"
+        />
+      </div>
+    `;
+
+    const content = html`
+      <div key="content" class="topic-teaser-content">
```

Comment:
You will find the `key` attribute in many old stories, but these are React attributes and therefore should be removed in new stories.

```suggestion
      <div class="topic-teaser-content">
```